### PR TITLE
Update author in header to match pdftitle wording

### PR DESCRIPTION
**Pull request by fulldecent**

### CODE REVIEW

File: Paper.tex [hunk 1]
```diff
@@ -64,8 +64,7 @@
 \title{Ethereum: A Secure Decentralised Generalised Transaction Ledger \\ {\smaller \textbf{Byzantium version \YellowPaperVersionNumber}}}
 \author{
     Dr. Gavin Wood\\
-    Founder, Ethereum \& Parity\\
-    gavin@parity.io
+    and others as per https://github.com/ethereum/yellowpaper/commits/master
```
Comment (jamesray1) [inline Paper.tex]: Just capitalise "and" and re-add the contact details.

File: Paper.tex [hunk 2]
```diff
@@ -66,6 +66,7 @@
     Dr. Gavin Wood\\
     Founder, Ethereum \& Parity\\
     gavin@parity.io
```
Comment (jamesray1) [inline Paper.tex]: I think this line also needs a \\

File: Paper.tex [hunk 3]
```diff
@@ -66,6 +66,7 @@
...
+    And others as per https://github.com/ethereum/yellowpaper/commits/master
```
Comment (jamesray1) [inline Paper.tex]: Please change this to `And others as detailed \href{https://github.com/ethereum/yellowpaper/graphs/contributors}{here}.`

**Comment by fulldecent** [comment]
Thank you @jamesray1 , updated

**Comment by nicksavers** [comment]
Git history is for all to see in the repository and some contributions are not done through Git. There is an acknowledgement section for those that made substantial contributions to the document.

**Comment by fulldecent** [comment]
Then say: "And contributors" and don't link to github.

In the past, one person did the writing and others are acknowledged. In modern times, many people do the writing. That's what we have here.

**Comment by pirapira** [comment]
I am maintaining Gavin's document.  You can fork your specification under the license.

**Comment by fulldecent** [comment]
You are correct, anybody can fork and share this.

My understanding is that when you accept other peoples' (non-trivial) pull requests into this document then they become authors. You don't have to name them all, that's why you just say "And contributors."

**Comment by pirapira** [comment]
Gavin should decide.  By default, I keep the current presentation.

**Comment by fulldecent** [comment]
Fair.

I'll leave it to your to bring this to Gavin's attention if you feel this is warranted.

And I respect your demonstrated integrity on other matters here, so I'll just leave it at that.

**Comment by jamesray1** [comment]
For the record, https://github.com/ethereum/yellowpaper/graphs/contributors works for me.

**Comment by pirapira** [comment]
The link in the PDF didn't work because some alphabets turned into capital letters.

**Comment by jamesray1** [comment]
Yeah I noticed that, I wonder how that happened as the diff doesn't have the ALLCAPs.

**Pull request closed without merging**